src/guides/v2.3/config-guide/revert-split-database.md

```diff
@@ -0,0 +1,72 @@
+---
+group: configuration-guide
+title: Configure a single database from a split database
+ee_only: true
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
+
+## Revert a split database implementation
+
+Reverting from a split database to a single database implementation involves creating backups of the `magento_quote` and `magento_sales` databases before loading them into the single `magento_main` database.
+
+In this example, all three databases are installed on the same host named `magento2-mysql`.
```

Comment:
It is worth mentioning that in the examples below "root" MySQL user is used, but it has to be replaced with the appropriate user name for the Magento instance.